server/command.go

```diff
@@ -366,6 +375,20 @@ func (p *Plugin) SendLinksWithDetails(userID, channelID string, links []*storemo
 	p.sendBotEphemeralPost(userID, channelID, sb.String())
 }
 
+func (p *Plugin) ShowConnectedUsers(userID, channelID string, connectedUsers []*storemodels.ConnectedUsers) {
+	var sb strings.Builder
+	sb.WriteString("| Mattermost User ID | MS Teams User ID | \n|:-------|:-----------|")
+	for _, connectedUser := range connectedUsers {
+		row := fmt.Sprintf("\n|%s|%s|", connectedUser.MattermostUserID, connectedUser.TeamsUserID)
+
+		if row != "\n|||" {
```

Comment:
if one of the IDs is empty then also we will display the result?

Reply:
If this case is never possible then also I think it will be more logical to add a check for IDs != "" instead of rox != "\n|||". What do you think?